backend/src/routes/workflow.py

```
@@ -1,11 +1,13 @@
 from flask import Blueprint, request, jsonify
+from flask_jwt_extended import jwt_required
```

Pylint (3.3.7)

[error] Unable to import 'flask_jwt_extended'

(E0401)